src/SpecflowAutomationTest/Calculator.feature

```diff
@@ -1,6 +1,88 @@
-﻿Feature: Calculator
+Feature: Calculator
 	Simple calculator for applying math operators on two numbers
 
 	#Please define and implement ALL needed scenarios for testing a simple calculator.
 	#The calculatoor has only one feature and it is to apply math operations on two numbers.
-	#The calculator is located at SampleLib project and has a method called "Calculate".
\ No newline at end of file
+	#The calculator is located at SampleLib project and has a method called "Calculate".
+
+#basic scenarios
+  
+ Scenario: Add two numbers
```

Comment:
I don't see a case testing division by zero.
Can't see a case testing the IsSuccessful = false.
Nothing for the exception logging.